### PR TITLE
refactor(web-app): replace spacing tokens with rem function for height and width

### LINKED ISSUE #572: Update spacing tokens for just spacing and use rem mixin for heights/widths
### Summary

- [x] Update spacing tokens throughout the app to follow Carbon guidelines (i.e. remove spacing tokens from unnecessary uses - width & height)
- [x] Include rem mixin

For context: https://ibm-studios.slack.com/archives/C02H44VRGQP/p1649883085903599

### Justification

_No response_

### Success metrics

_No response_

### Required functionality

_No response_

### Specific timeline issues / requests

_No response_

### Available resources

_No response_

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #572 

#### Changelog

**Changed**

- Update all height/width pixel values to use rem function
- Convert spacing tokens to pixel values for height/width

#### Testing / reviewing
 
Nothing should visually change 🙏

### CODE REVIEW

**Review by francinelucca — CHANGES_REQUESTED** [review]
Also looks like we're missing some on:
- services/web-app/components/aside/aside.module.scss (L44)
- services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.module.scss (L102,106,142)
- services/web-app/components/color-palette/color-palette.module.scss (L167)
- services/web-app/components/color-token-table/color-token-table.module.scss (L31)
- services/web-app/components/svg-libraries/svg-library.module.scss (L54, 58,96,302)
- services/web-app/pages/assets/index/index.module.scss (L26)
- services/web-app/styles/vendor/_overrides.scss (L35)

File: services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.module.scss [hunk 1]
```diff
@@ -80,7 +81,7 @@
   top: spacing.$spacing-01;
   right: spacing.$spacing-01;
   display: inline-block;
-  min-width: spacing.$spacing-05 + spacing.$spacing-02;
+  min-width: convert.rem(18px);
```
[the remark this answers is not in the document]
Comment (alisonjoseph) [reply services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.module.scss:84]: yep!

**Comment by alisonjoseph** [comment]
@francinelucca thanks! Fixed the missing values, also left anything at 1px that was a border. (for example L26 in /assets/index/index.module.scss )

**Review by francinelucca — APPROVED** [review]
LGTM! 💃 ✨

**Review by andreancardona — COMMENTED** [review]
How do we feel about the page header height in `page-header.module.scss`? Should this be converted as well?

Screen Shot 2022-05-15 at 10 32 33 AM

**Comment by alisonjoseph** [comment]
@andreancardona that line was removed in #715 that was just merged, I'll update the branch and double check things.

**Review by andreancardona — APPROVED** [review]
🔥

**Pull request merged**